src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -24,6 +24,18 @@ public partial class ShadowContainer
 	private ShadowPaintState? _lastPaintState;
 	private bool _isShadowDirty;
 
+	public event EventHandler<SurfacePaintCompletedEventArgs>? SurfacePaintCompleted;
```

Comment:
These should be internal, but I'm they cant be accessed by the test project even thought the project has [assembly: InternalsVisibleTo("Uno.Toolkit.RuntimeTests")]

Reply:
i added to src/Uno.Toolkit.Skia.WinUI/AssemblyInfo.cs project